js/data.js

```diff
@@ -15,3 +17,13 @@ var previousInput = localStorage.getItem('data-model');
 if (previousInput !== null) {
   data = JSON.parse(previousInput);
 }
+
+function dataView() {
+  var viewElements = document.querySelectorAll('.view');
+  for (var i = 0; i < viewElements.length; i++) {
+    if (viewElements[i].className === 'container view') {
+      data.view = viewElements[i].getAttribute('data-view');
+    }
+  }
+  return data.view;
+}
```

Comment:
Instead of doing this, you should just assign a new value to `data.view` every time a user moves to a new view in your code, such as when the form is submitted, a link is clicked, etc. Then you can use your `assignClassNames` function (which I would rename to `switchView`) to change the views every time a new view is set. That will remove the need for a lot of the repetitious functions you have for moving between views.